rules-reviewed/eap7/eap72/maven-jboss-rmi-api_1.0_spec.rhamt.xml

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0"?>
+<ruleset xmlns="http://windup.jboss.org/schema/jboss-ruleset" id="maven-jboss-rmi-api_1.0_spec"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://windup.jboss.org/schema/jboss-ruleset http://windup.jboss.org/schema/jboss-ruleset/windup-jboss-ruleset.xsd">
+    <metadata>
+        <description>
+            This ruleset provides analysis of Maven built applications that depend on org.jboss.spec.javax.rmi:jboss-rmi-api_1.0_spec artifact, a dependency that should be removed when migrating to JBoss EAP 7.3 or later.
+        </description>
+        <dependencies>
+            <addon id="org.jboss.windup.rules,windup-rules-javaee,3.0.0.Final" />
+            <addon id="org.jboss.windup.rules,windup-rules-java,3.0.0.Final" />
+        </dependencies>
+        <targetTechnology id="eap" versionRange="[7,8)" />        
+    </metadata>
+    <rules>
+        <!-- https://issues.jboss.org/browse/WINDUPRULE-401 -->
+        <rule id="maven-jboss-rmi-api_1.0_spec-00001">
+            <when>
+                <project>
+                    <artifact groupId="org.jboss.spec.javax.rmi" artifactId="jboss-rmi-api_1.0_spec" />
+                </project>
+            </when>
+            <perform>
+                <hint title="Remove Maven dependency on org.jboss.spec.javax.rmi:jboss-rmi-api_1.0_spec" effort="1" category-id="potential">
+                    <message>If you migrate your application to JBoss EAP 7.3, and want to ensure its Maven building, running or testing works as expected, remove any dependency on artifact with groupId `org.jboss.spec.javax.rmi`, and artifactId `jboss-rmi-api_1.0_spec`.</message>
+                    <link title="FIXME" href="FIXME"/>
```

Comment:
@emmartins i think the idea for this rule is to have it without a link since the change is very basic and well explained in hint's message. Right?
If yes, please remove this `<link>` tag.

Reply:
done